app/controllers/issues_controller.rb

```diff
@@ -32,6 +32,11 @@ def show
     @first_evidence  = Evidence.where(node: @first_node, issue: @issue)
 
     load_conflicting_revisions(@issue)
+
+    @subscription = Subscription.find_by(
```

Comment:
Can we shorten this with the syntax: `@issue.subscriptions.where(user: current_user)`?

Reply:
I was trying to avoid using something like this: https://github.com/dradis/dradis-ce/blob/master/app/models/issue.rb#L19